cmd/regula/cli/cli.go

```diff
@@ -25,8 +25,9 @@ import (
 // Config holds the server configuration.
 type Config struct {
 	Etcd struct {
-		Endpoints []string `config:"etcd-endpoints"`
-		Namespace string   `config:"etcd-namespace,required"`
+		Endpoints   []string      `config:"etcd-endpoints"`
+		Namespace   string        `config:"etcd-namespace,required"`
+		DialTimeout time.Duration `config:"etcd-dial-timemout"`
```

Comment:
Is this change strictly necessary / related?

Reply:
It is not related at all actually, it was just in my way, to be honest. About the necessity of this change, I thought it would be useful to be able to control how long someone accepts to wait for etcd to be ready I can remove it though, WDYT?